app/src/main/java/com/infomaniak/mail/ui/main/thread/actions/ThreadActionsBottomSheetDialog.kt

```diff
@@ -96,6 +98,10 @@ class ThreadActionsBottomSheetDialog : ActionsBottomSheetDialog() {
                     mainViewModel.toggleFavoriteStatus(threadUid)
                 }
 
+                override fun onSpam() {
+                    mainViewModel.toggleSpamOrHam(threadUid)
+                }
+
                 override fun onReportJunk() {
                     safeNavigate(
```

Comment:
This shouldn't exist anymore, right?
But remove it in the same PR as this https://github.com/Infomaniak/android-mail/pull/593/files#r1108241640